makefile.shade

```diff
@@ -296,13 +296,19 @@ var buildTarget = "compile"
         }
     }
 
-#cache-coherence
+#use-local-coherence description='Sets up environment variables to use Coherence at COHERENCE_PATH'
     @{
         if (string.IsNullOrEmpty(coherencePath))
         {
             throw new Exception("COHERENCE_PATH not specified.");
         }
 
+        universeCommitsFile = Path.Combine(coherencePath, UniverseCommitsFileName);
```

Comment:
This overrides the first value set if the the `UNIVERSE_COMMITS_FILE` environment variable was set. Check if it's null/empty before doing this.